content/docs/warehouses-supported/bigquery.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Google BigQuery
+---
+
+# Authentification
+
+Dataform will connect to BigQuery using a service account. You’ll need to create a service account from your Google Cloud Console and assign it permissions to access BigQuery.
+
+1. Follow <a target="_blank" rel="noopener" href="https://cloud.google.com/iam/docs/creating-managing-service-accounts#creating_a_service_account">these instructions</a> to create a new service account in Google Cloud Console.
+2. Grant the new account the `BigQuery Admin` role. (Admin access is required by Dataform so that it can create queries and list tables.) Read
+   <a target="_blank" rel="noopener" href="https://cloud.google.com/iam/docs/granting-roles-to-service-accounts#granting_access_to_a_service_account_for_a_resource">this</a> if you need help.
+3. Create a key for your new service account (in JSON format). You will upload this file to Dataform. Read
+   <a target="_blank" rel="noopener" href="https://cloud.google.com/iam/docs/creating-managing-service-account-keys#creating_service_account_keys">this</a> if you need help.
+
+BigQuery specific options can be applied to tables using the `bigquery` configuration parameter.
+
+# Configuration options
+
+## Setting table partitions
+
+BigQuery supports <a target="_blank" rel="noopener" href="https://cloud.google.com/bigquery/docs/partitioned-tables">partitioned tables</a>.
+These can be useful when you have data spread across many different dates but usually query the table on only a small range of dates.
+In these circumstances, partitioning will increase query performance and reduce cost.
+
+BigQuery partitions can be configured in Dataform using the `partitionBy` option:
+
+```js
+config {
+  type: "table",
+  bigquery: {
+    partitionBy: "DATE(ts)"
+  }
+}
+SELECT CURRENT_TIMESTAMP() AS ts
+```
+
+This query compiles to the following statement which takes advantage of BigQuery's DDL to configure partitioning:
+
+```js
+CREATE OR REPLACE TABLE dataform.example
+PARTITION BY DATE(ts)
+AS (SELECT CURRENT_TIMESTAMP() AS ts)
+```
+
+If desired, partitions can be clustered by using the `clusterBy` option, for example:
```

Comment:
I'd probably make this a separate section, rather than rolling it under the partitioning section, as I think it'll make it easer for users to find the information